stl/inc/yvals_core.h

```diff
@@ -799,7 +799,21 @@
 #define _DEPRECATE_STDEXT_HASH_UPPER_BOUND
 #endif // ^^^ warning disabled ^^^
 

```

Comment:
You're adding a new feature, so you should update `_MSVC_STL_UPDATE` to October. (We should consider automating this, somehow.)